internal/jennies/golang/rawtypes.go

```diff
@@ -147,6 +142,154 @@ func (jenny RawTypes) formatObject(schema *ast.Schema, def ast.Object) ([]byte,
 			buffer.WriteString("}\n")
 		}
 	}
+}
+
+func (jenny RawTypes) generateConstructor(buffer *strings.Builder, context languages.Context, object ast.Object) {
+	defName := formatObjectName(object.Name)
```

Comment:
JooC, any reason for not calling this `objName` ?

Reply:
No good reason: objects used to be called "definitions" a long time ago, and old habits die hard :sweat_smile: